main.rb

```diff
@@ -1,12 +1,61 @@
-require_relative('./person')
-require_relative('./trimmer')
-require_relative('./capitalize')
-require_relative('./nameable')
-require_relative('./decorator')
-p person = Person.new(22, 'maximilianus')
-p person.correct_name
-# capitalizedPerson = Capitalize.new(person)
-# p capitalizedPerson
-# p capitalizedPerson.correct_name
-# capitalizedTrimmedPerson = Trimmer.new(capitalizedPerson)
-# p capitalizedTrimmedPerson.correct_name
+require './app'
+
+class Main
+  attr_reader :app
+
+  def initialize
+    puts 'School library application'
+    @app = App.new
+    item_list
+  end
+
+  def item_list
+    puts '================================'
+    puts 'Choose an option:'
+    puts '================================'
+    puts "
+  1 - List all books
+  2 - List all people
+  3 - Create a person
+  4 - Create a book
+  5 - Create a rental
+  6 - List all rentals for a given person id
+  7 - Exit"
+    selection_user
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] Rather than disable the Linters, strive to fix them as this makes you more accustomed to the style guide of the tech.